neo/rawio/maxwellrawio.py

```diff
@@ -0,0 +1,221 @@
+"""
+Class for reading data from maxwell biosystem device:
+  * MaxOne
+  * MaxTwo
+
+https://www.mxwbio.com/resources/mea/
+
+The implementation is a mix between:
+  * the implementation in spikeextractors
+     https://github.com/SpikeInterface/spikeextractors/blob/master/spikeextractors/extractors/maxwellextractors/maxwellextractors.py
+ * the implementation in spyking-circus
+    https://github.com/spyking-circus/spyking-circus/blob/master/circus/files/maxwell.py
+
+The implementation do not handle spike at the moment.
+
+For maxtwo device, each well will be a different signal stream.
+
+Author : Samuel Garcia, Alessio Buccino, Pierre Yger
+"""
+import os
+from pathlib import Path
+import platform
+from urllib.request import urlopen
+
+from .baserawio import (BaseRawIO, _signal_channel_dtype, _signal_stream_dtype,
+                _spike_channel_dtype, _event_channel_dtype)
+
+import numpy as np
+
+try:
+    import h5py
+    HAVE_H5 = True
+except ImportError:
+    HAVE_H5 = False
+
+
+class MaxwellRawIO(BaseRawIO):
+    """
+    Class for reading MaxOne or MaxTwo files.
+    """
+    extensions = ['h5']
+    rawmode = 'one-file'
+
+    def __init__(self, filename='', rec_name=None):
+        BaseRawIO.__init__(self)
+        self.filename = filename
+        self.rec_name = rec_name
+
+    def _source_name(self):
+        return self.filename
+
+    def _parse_header(self):
+        try:
+            import MEArec as mr
+            HAVE_MEAREC = True
+        except ImportError:
+            HAVE_MEAREC = False
+        assert HAVE_H5, 'h5py is not installed'
+
+        h5 = h5py.File(self.filename, mode='r')
+        self.h5_file = h5
+        version = h5['version'][0].decode()
+
+        # create signal stream
+        # one stream per well
+        signal_streams = []
+        if int(version) == 20160704:
```

Comment:
what should happen in case of `int(version) < 20160704` ? Are you sure no older versions exist?

Reply:
yes this is the first released version :)

Reply:
@JuliaSprenger maybe we can add an `else: raise Exception("Unsupported version")` for completeness?

Reply:
yes, sounds good, just to be on the save side.

Reply:
@samuelgarcia wdyt?